src/mongocxx/options/server_api.hpp

```diff
@@ -0,0 +1,139 @@
+// Copyright 2020 MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <mongocxx/config/prelude.hpp>
```

Comment:
Move to after driver headers

Reply:
The .clang-format has a macro enforcing `prelude.hpp` appears before other headers:
https://github.com/mongodb/mongo-cxx-driver/blob/908f0e6de24ed60cb983726d11c6aed5ecee3922/.clang-format#L12

I vote we revert this so the `lint` task passes for now, and fix them all, with the .clang-format file, as part of CXX-2258.